govcd/org.go

```diff
@@ -215,16 +215,18 @@ func (org *Org) GetCatalogByHref(catalogHref string) (*Catalog, error) {
 // GetCatalogByName  finds a Catalog by Name
 // On success, returns a pointer to the Catalog structure and a nil error
 // On failure, returns a nil pointer and an error
+//
+// refresh has no effect here, but is kept to preserve signature
 func (org *Org) GetCatalogByName(catalogName string, refresh bool) (*Catalog, error) {
-	if refresh {
-		err := org.Refresh()
-		if err != nil {
-			return nil, err
-		}
+
+	orgCatalogs, err := org.QueryCatalogList()
```

Comment:
it would be easier/faster to query catalog by name(using filter)

Reply:
It makes sense and I changed it.